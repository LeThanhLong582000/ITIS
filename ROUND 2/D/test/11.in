7
5
11
31
1
19
49
11
