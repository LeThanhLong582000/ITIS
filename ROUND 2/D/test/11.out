15
773
387559437
1
147312
52298426843184
773
